config/redshift.config.reference.hocon

```diff
@@ -138,6 +138,15 @@
     "cumulativeBound": "1 hour"
   },
 
+  # Check the target destination to make sure it is ready.
+  # Retry the checking until target got ready and block the application in the meantime
+  "readyCheck": {
+    # Starting backoff period
+    "backoff": "60 seconds"
```

Comment:
I think there's nothing wrong if we provide more granular defaults, like 5 seconds. I.e. the `SELECT 1` isn't a big deal performance-wise, but imagine if DB becomes available after 61 seconds - it means there will be a whole 59 seconds delay for loading, which is quite a lot.